test/conversion.test.js

```diff
@@ -356,4 +356,50 @@ describe('Curl converter should', function() {
     expect(header.value).to.equal('$v1');
     done();
   });
+
+  it('[GitHub #8126] [GitHub #7983] [GitHub #7895]: should import body data with --data-raw argument', function (done) {
+
+    // #8126 content-type application/json, so mode = raw
+    var result = Converter.convertCurlToRequest(`curl --location --request PUT
```

Comment:
@shreys7 we don't need the exact request the user reported. The test case should be as small as possible.

Reply:
Okay. Will change the tests.